discovery_engine_core/providers/src/client.rs

```diff
@@ -461,19 +443,19 @@ mod tests {
         assert_eq!(docs.len(), 2);
 
         let doc = docs.get(1).unwrap();
-        let expected = Article {
+        let expected: GenericArticle = crate::newscatcher::Article {
```

Comment:
🔧 Here I would also prefer to create a `GenericArticle` directly so that we don't depend on the newscatcher article